app/controllers/user_controller.py

```diff
@@ -0,0 +1,19 @@
+from typing import Any
+from flask import Blueprint, render_template, session
+from app.models import User
+from app.utils.session_decorators import session_required
+
+
+user_routes = Blueprint('user_routes', __name__)
+
+
+@user_routes.route('/<username>', methods=['GET'])
+@session_required
+def get_user(username) -> Any:
```

Comment:
This line needs to accept 2 parameters or it will throw an exception, because the `@session_required` wrapper tries to inject the user_id into the function, so it needs to be
`def get_user(user_id, username) -> Any:`